File: android/versioned-abis/expoview-abi39_0_0/src/main/java/abi39_0_0/expo/modules/sms/SMSPackage.java

package abi39_0_0.expo.modules.sms;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi39_0_0.org.unimodules.core.BasePackage;
import abi39_0_0.org.unimodules.core.ExportedModule;
import abi39_0_0.org.unimodules.core.interfaces.InternalModule;

public class SMSPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context reactContext) {
    return Collections.singletonList((ExportedModule) new SMSModule(reactContext));
  }
}

File: android/versioned-abis/expoview-abi43_0_0/src/main/java/abi43_0_0/expo/modules/sms/SMSPackage.kt
package abi43_0_0.expo.modules.sms

import android.content.Context
import abi43_0_0.expo.modules.core.BasePackage
import abi43_0_0.expo.modules.core.ExportedModule

class SMSPackage : BasePackage() {
  override fun createExportedModules(reactContext: Context): List<ExportedModule> {
    return listOf(SMSModule(reactContext))
  }
}
